lambda_functions/ml_service/EnrichItem.py

```diff
@@ -280,6 +281,68 @@ def store_itementities(event, context, is_test=False, session=None):
                 raise
 
 
+def store_itemtags(event, context, is_test=False, session=None):
+    """stores tags of an item
+
+    Parameters
+    ----------
+    event: dict, required
+        item
+        Tags
+
+        Event doc: https://docs.aws.amazon.com/apigateway/latest/developerguide/set-up-lambda-proxy-integrations.html#api-gateway-simple-proxy-for-lambda-input-format
+
+    context: object, required
+        Lambda Context runtime methods and attributes
+
+        Context doc: https://docs.aws.amazon.com/lambda/latest/dg/python-context-object.html
+
+    Returns
+    ------
+    API Gateway Lambda Proxy Output Format: application/json
+
+        Return doc: https://docs.aws.amazon.com/apigateway/latest/developerguide/set-up-lambda-proxy-integrations.html
+    """
+
+    if session is None:
+        session = get_db_session(is_test, session)
+
+    # Store all entities of the item
+    for str_tag in event['Tags']:
+        tag = Tag()
+        # search for tag in database
+        try:
+            tag = tag_handler.get_tag_by_content(
+                str_tag, is_test, session)
+        except Exception:
+            # store tag in database
+            tag.id = str(uuid4())
+            tag.tag = str_tag
+            try:
+                update_object(tag, is_test, session)
+            except Exception as e:
+                logger.error(
+                    "Could not store tag. Exception: %s", e, exc_info=True)
+                raise
+        # store item tag in database
+        itemtag = ItemTag()
+        # item tag already exists?
+        item_id = event['item']['id']
+        try:
+            itemtag = tag_handler.get_itemtag_by_tag_and_item_id(
+                tag.id, item_id, is_test, session)
+        except Exception:
+            itemtag.id = str(uuid4())
+            itemtag.item_id = item_id
+            itemtag.tag_id = tag.id
+            try:
+                update_object(itemtag, is_test, session)
+            except Exception as e:
+                logger.error(
+                    "Could not store item tag. Exception: %s", e, exc_info=True)
+                raise
```

Comment:
See previous comment

Reply:
get_itemtag_by_tag_and_item_id now returns None if the itemtag is not existing